src/lib/src/component/navigation/navigation.component.spec.ts

```diff
@@ -33,13 +33,20 @@ describe('NavigationComponent', () => {
   let spy: jasmine.Spy;
   let sidenav: DebugElement;
 
-  let listedFile: WorkspaceElement = {
```

Comment:
Just for the future, renaming would be better suited for a separate pull. Makes reviewing much easier